core/src/com/group66/game/cannon/BallManager.java

```diff
@@ -314,18 +314,18 @@ public boolean isGameComplete() {
      * @param batch the batch used to draw with
      * @param delta the delta
      */
-    public void draw(SpriteBatch batch, float delta) {
+    public void draw(AbstractGameScreen gameScreen, SpriteBatch batch, float delta) {
         int xoffset = Config.SINGLE_PLAYER_OFFSET;
         if (isSplit) {
             xoffset = Config.SEGMENT_OFFSET * segmentOffset;
         }
         
         //draw the background
-        batch.draw(AssetLoader.bg, xoffset + Config.BORDER_SIZE_SIDES, Config.BORDER_SIZE_BOT,
+        batch.draw(gameScreen.bg, xoffset + Config.BORDER_SIZE_SIDES, Config.BORDER_SIZE_BOT,
```

Comment:
bg is a public static attribute. you should access it in a static way or change the way the texture is retrieved from the AbstractGameScreen